<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RhapsodyImportProject</class>
 <widget class="QDialog" name="RhapsodyImportProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import from Rhapsody</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0">
     <item row="0" column="0" alignment="Qt::AlignBottom">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Choose a Rhapsody project:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignVCenter">
      <widget class="QLineEdit" name="projectFileBrowser">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QPushButton" name="browsePushButton">
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignBottom">
      <widget class="QPushButton" name="clearPushButton">
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="toolTip">
        <string extracomment="No active instance or project found. Click to check again"/>
       </property>
       <property name="toolTipDuration">
        <number>5</number>
       </property>
       <property name="text">
        <string>Load from active Rhapsody instance</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="refreshPushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="qmlRhapsodyExample.qrc">
         <normaloff>:/qmlRhapsodyExample/images/refresh.png</normaloff>:/qmlRhapsodyExample/images/refresh.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="cancelPushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qmlRhapsodyExample.qrc"/>
 </resources>
 <connections/>
</ui>
